config/runtimes/vllm-rocm-template.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: vLLM ROCM is a high-throughput and memory-efficient inference and serving engine for LLMs
+    openshift.io/display-name: vLLM ROCM ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/vllm
+    template.openshift.io/long-description: This template defines resources needed to deploy vLLM ServingRuntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: 'REST'
+  name: vllm-rocm-runtime-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: vllm-rocm-runtime
+      annotations:
+        openshift.io/display-name: vLLM ROCM ServingRuntime for KServe
+        opendatahub.io/recommended-accelerators: '["amd.com/gpu"]'
+      labels:
+        opendatahub.io/dashboard: 'true'
+    spec:
+      annotations:
+        prometheus.io/port: '8080'
+        prometheus.io/path: '/metrics'
+      multiModel: false
+      supportedModelFormats:
+        - autoSelect: true
+          name: vLLM
+      containers:
+        - name: kserve-container
+          image: $(vllm-rocm-image)
+          command:
+            - python
+            - -m
+            - vllm.entrypoints.openai.api_server
+          args:
+            - "--port=8080"
+            - "--model=/mnt/models"
+            - "--served-model-name={{.Name}}"
+          env:
+            - name: HF_HOME
+              value: /tmp/hf_home
+          ports:
+            - containerPort: 8080
+              protocol: TCP
```

Comment:
no resources and limits?

Reply:
Resources and limits would be provided through the InferenceService.

Reply:
Can we provide the `resources` as a back-up, for users using GitOps or YAML?